app/views/second_level_browse_page/_links.html.erb

```diff
@@ -1,5 +1,6 @@
 <% track_action ||= "contentLink" %>
 
+<% total_links = list.count.to_s %>
 <ul class="govuk-list browse__list" data-module="gem-track-click">
```

Comment:
I think we might need to add `ga4-link-tracker` here

Reply:
Started checking this and was getting more and more confused as I couldn't find the tracking on this level or any level of browse, thought I was losing my mind until I noticed I was looking at the live site and not my local 🤦 

Have added the module to a higher level in `second_level_browse_page/show_a_to_z.html.erb` and `second_level_browse_page/show_curated.html.erb` rather than adding it multiple times throughout the child elements.

Thanks for finding this!